emlparser/emlparser.py

```diff
@@ -69,13 +69,14 @@ def execute(self, request):
             kv_section = ResultSection('Email Headers', body_format=BODY_FORMAT.KEY_VALUE, parent=result)
 
             # Basic tags
-            if header.get('from', None):
+            if header.get('from', None) and re.match(EMAIL_REGEX, header['from'].strip()):
                 kv_section.add_tag("network.email.address", header['from'].strip())
             [kv_section.add_tag("network.email.address", to.strip())
              for to in header['to'] if re.match(EMAIL_REGEX, to.strip())]
 
             kv_section.add_tag("network.email.date", str(header['date']).strip())
-            kv_section.add_tag("network.email.subject", header['subject'].strip())
+            if header['subject'].strip():
```

Comment:
rather than performing the strip method twice, assign it to a variable and use that